static/usage/v7/item/content-types/text/vue.md

```diff
@@ -0,0 +1,74 @@
+```html
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>Example</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content color="light">
+    <ion-list :inset="true">
+      <ion-item>
+        <ion-input label="First Name"></ion-input>
+      </ion-item>
+      <ion-item>
+        <ion-input label="Last Name"></ion-input>
+      </ion-item>
+      <ion-item>
+        <ion-toggle>
+          <ion-label>Enable Notifications?</ion-label>
```

Comment:
```suggestion
          <ion-label>Allow Notifications</ion-label>
```

This one got added back on accident I think: https://github.com/ionic-team/ionic-docs/commit/0f6f3f2081a98cb440e40a20a2c7da8d36da57ca